src/plugins/janus_sip.c

```diff
@@ -1047,6 +1047,7 @@ typedef struct janus_sip_media {
 	gboolean updated;
 	int video_orientation_extension_id;
 	int audio_level_extension_id;
+	int dtmf_pt;
```

Comment:
Any field you add to `janus_sip_media` must be reset along the others in `janus_nosip_media_reset`, which your patch is currently not doing. Please add a reset to `-1` in there.